addons/l10n_id_efaktur_coretax/models/account_move.py

```diff
@@ -0,0 +1,317 @@
+        ('TD.00509', '9 - PPN DIBEBASKAN BERDASARKAN PP NOMOR 47 TAHUN 2020'),
+        ('TD.00510', '10 - PPN Dibebaskan berdasarkan PP Nomor 49 Tahun 2022'),
+    ])
+
+    l10n_id_facility_info_08 = fields.Selection([
+        ('TD.01101', '1 - PPN Dibebaskan Sesuai PP Nomor 146 Tahun 2000 Sebagaimana Telah Diubah Dengan PP Nomor 38 Tahun 2003'),
+        ('TD.01102', '2 - PPN Dibebaskan Sesuai PP Nomor 12 Tahun 2001 Sebagaimana Telah Beberapa Kali Diubah Terakhir Dengan PP Nomor 31 Tahun 2007'),
+        ('TD.01103', '3 - PPN dibebaskan berdasarkan Peraturan Pemerintah Nomor 28 Tahun 2009'),
+        ('TD.01104', '4 - (Tidak ada cap)'),
+        ('TD.01105', '5 - PPN Dibebaskan Sesuai Dengan PP Nomor 81 Tahun 2015'),
+        ('TD.01106', '6 - PPN Dibebaskan Berdasarkan PP Nomor 74 Tahun 2015'),
+        ('TD.01107', '7 - (tanpa cap)'),
+        ('TD.01108', '8 - PPN DIBEBASKAN SESUAI PP NOMOR 81 TAHUN 2015 SEBAGAIMANA TELAH DIUBAH DENGAN PP 48 TAHUN 2020'),
+        ('TD.01109', '9 - PPN DIBEBASKAN BERDASARKAN PP NOMOR 47 TAHUN 2020'),
+        ('TD.01110', '10 - PPN Dibebaskan berdasarkan PP Nomor 49 Tahun 2022'),
+    ])
+
+    l10n_id_kode_transaksi = fields.Selection(selection_add=[('10', '10 Penyerahan lainnya')])
+    l10n_id_add_info = fields.Char()
+    l10n_id_stlg_rate = fields.Float(string="STLG Rate", default=0, copy=False)  # luxury good rate
+    l10n_id_efaktur_available = fields.Boolean(compute="_compute_l10n_id_efaktur_available")
+
+    l10n_id_efaktur_document = fields.Many2one('l10n_id_efaktur_coretax.document', readonly=True, copy=False, string="e-Faktur Document")
+
+    def _compute_need_kode_transaksi(self):
+        """ OVERRIDE: l10n_id_efaktur
+
+        By setting this l10n_id_need_kode_transaksi, we can prevent the old E-Faktur flow to be
+        triggered(i.e. efaktur range consumption).
+        """
+        self.l10n_id_need_kode_transaksi = False
+
+    @api.depends('partner_id.l10n_id_pkp', 'line_ids.tax_ids')
+    def _compute_l10n_id_efaktur_available(self):
+        """ Similar use case as l10n_id_need_kode_transaksi from l10n_id_efaktur
+
+        helps to check whether or not some fields need to be visible or not
+        """
+        for move in self:
+            move.l10n_id_efaktur_available = (
+                move.partner_id.l10n_id_pkp
+                and move.country_code == 'ID'
+                and move.move_type == 'out_invoice'
+                and move.line_ids.tax_ids
+            )
+
+    # Interactions that is triggered when choosing kode 07 or 08
+    # as they require additional information
+    @api.onchange("l10n_id_add_info_07")
+    def _onchange_l10n_id_add_info_07(self):
+        if not self.l10n_id_add_info_07:
+            self.l10n_id_facility_info_07 = False
+        else:
+            digit = self.l10n_id_add_info_07[-2:]
+            self.l10n_id_facility_info_07 = 'TD.011' + digit
```

Comment:
It's best to use a computed editable field; this way, it's triggered even from a test/...